### ISSUE TITLE
getTransactionId() should this return req_reference_number?

### ISSUE THREAD

**Issue by veganista** [PatronBase/omnipay-cybersource-hosted#3]
I'm wondering if this package should implement the getTransactionId method and return the `req_reference_number`.

This is useful for looking the payment from your own database reference.

Currently to get this value you need to do (in laravel anyway):

```php
$request->input('req_reference_number')
```

But it sounds to me like you should be able to do this:

```php
$notification->getTransactionId()
```

The omnipay docs show this:

```php
    /**
     * Gateway Reference
     *
     * @return null|string A reference provided by the gateway to represent this transaction
     */
    public function getTransactionReference()
    {
        return null;
    }

    /**
     * Get the transaction ID as generated by the merchant website.
     *
     * @return string
     */
    public function getTransactionId()
    {
        return null;
    }

```

This cybersource package implements it like so:

https://github.com/dioscouri/omnipay-cybersource/blob/1f59de5d0eee27a3e021a61b7ad8e13b4afb7ea4/src/Message/CompletePurchaseResponse.php#L17

**Comment by leith** [comment]
At the moment rather than looking at the request you could use `getData()` from the message object, but I don't see why that can't be built in. @veganista did you want to open a PR?

**Comment by leith** [comment]
Actually, this was pretty small so just went ahead and did it. 😅  This is available now in [v3.0.1](https://github.com/PatronBase/omnipay-cybersource-hosted/releases/tag/v3.0.1) & [v2.0.3](https://github.com/PatronBase/omnipay-cybersource-hosted/releases/tag/v2.0.3).
